test-harness/index.js

```diff
@@ -0,0 +1,270 @@
+const fetch = require('node-fetch');
+const requests = require('./requests');
+
+const { exec } = require('child_process');
+const { makeRequest, constructMasterFileName, readFile } = require('./helpers');
+
+async function waitForPrism(done) {
+  try {
+    await fetch('http://localhost:4010');
```

Comment:
If you're listening on `4010`, send this parameter explicitly when spawning Prism so we do not get surprises in case the port gets changed for any reason.

Reply:
Cool, how do I tell prism to start on port X?

Reply:
It is documented in the CLI page. The main Readme file has a link. https://github.com/stoplightio/prism/blob/master/packages/cli/README.md

I have no problems to add it to the main file as well though!